### ISSUE TITLE
[Feature Request] killed-policy

### ISSUE THREAD

**Issue by MartinThoma** [boxed/mutmut#248]
I would like to have `mutmut junitxml` only report the mutants that survived, hence I want to ignore the killed mutants. So `--killed-policy=ingore` would be very handy to have.

The reason why I would like to have that is two fold:

1. `mutmut junitxml` takes a lot of time to run
2. The html report generated by `junit2html mutmut-results.xml mutmut-results.html` is extremely long

**Comment by boxed** [comment]
Makes sense to me. I always thought it was a bit weird/useless to show killed mutants.

**Comment by Otto-AA** [comment]
Closing, as `mutmut junitxml` does not exist anymore (#344 )
